schema/Figure.schema.yaml

```diff
@@ -0,0 +1,28 @@
+title: Figure
+'@id': stencila:Figure
+extends: CreativeWork
+role: primary
+status: experimental
+category: prose
+description: Encapsulates one or more images, videos, tables, etc, and provides captions and labels for them.
+properties:
+  label:
+    '@id': stencila:label
+    description: A short label for the figure.
+    type: string
+  caption:
```

Comment:
When decoding JATS `<table>` we store the `<caption>` child element in the `CreativeWork.content` property. I wondering if we take the same approach here. In other words, do we need a `Figure.caption` property when we already have a `CreativeWork.content` property?